ghcide/src/Development/IDE/Spans/Documentation.hs

```diff
@@ -71,23 +70,21 @@ lookupKind :: HscEnv -> Module -> Name -> IO (Maybe TyThing)
 lookupKind env mod =
     fmap (fromRight Nothing) . catchSrcErrors (hsc_dflags env) "span" . lookupName env mod
 
-getDocumentationTryGhc :: HscEnv -> Module -> [ParsedModule] -> Name -> IO SpanDoc
-getDocumentationTryGhc env mod deps n = head <$> getDocumentationsTryGhc env mod deps [n]
+getDocumentationTryGhc :: HscEnv -> Module -> Name -> IO SpanDoc
+getDocumentationTryGhc env mod n = head <$> getDocumentationsTryGhc env mod [n]
 
-getDocumentationsTryGhc :: HscEnv -> Module -> [ParsedModule] -> [Name] -> IO [SpanDoc]
--- Interfaces are only generated for GHC >= 8.6.
--- In older versions, interface files do not embed Haddocks anyway
-getDocumentationsTryGhc env mod sources names = do
+getDocumentationsTryGhc :: HscEnv -> Module -> [Name] -> IO [SpanDoc]
+getDocumentationsTryGhc env mod names = do
   res <- catchSrcErrors (hsc_dflags env) "docs" $ getDocsBatch env mod names
   case res of
-      Left _ -> mapM mkSpanDocText names
+      Left _ -> return []
       Right res -> zipWithM unwrap res names
   where
     unwrap (Right (Just docs, _)) n = SpanDocString docs <$> getUris n
     unwrap _ n = mkSpanDocText n
 
     mkSpanDocText name =
-      SpanDocText (getDocumentation sources name) <$> getUris name
```

Comment:
`getDocumentation` can also be deleted.

Reply:
In fact, most of this file below this function can be deleted.

Reply:
No, `getDocumentation` is still used in the local completions code